### PR TITLE
#42: Add odbc driver tests

### LINKED ISSUE #42: Add JavaScript tests for ODBC connector
The ODBC connector does not yet have unit tests for the JavaScript code.

### PR DESCRIPTION
**Pull request by kaklakariada**
Closes #42

### CODE REVIEW

File: src/exasol_odbc/connectionBuilder.js [hunk 1]
```diff
@@ -1,22 +1,39 @@
...
+        log("ODBC Driver log enabled. Log file: " + odbcDriverLogFile);
+        params["EXALOGFILE"] = odbcDriverLogFile;
+        params["LOGMODE"] = "DEFAULT";
+    }
 
     params["EXAHOST"] = attr[connectionHelper.attributeServer];
     params["EXAPORT"] = attr[connectionHelper.attributePort];
 
-    if (attr["schema"] != "")
+    if (attr["schema"] && attr["schema"] != "") {
```
Comment (MarleneKress79789) [inline src/exasol_odbc/connectionBuilder.js:23]: Why the duplicate?

File: tools/run_tdvt_tests.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+project_dir="$( cd "$(dirname "$0")/.." >/dev/null 2>&1 ; pwd -P )"
+readonly project_dir
+target_dir="$project_dir/target/"
```
Comment (MarleneKress79789) [inline tools/run_tdvt_tests.sh:6]: Is this directory predefined? If not, maybe give it  mare specific name?

File: src/exasol_odbc/connectionBuilder.js (hunk 1 above)
Comment (kaklakariada) [reply src/exasol_odbc/connectionBuilder.js:23]: This handles the case when `attr["schema"]` is `undefined`. Without this, the output would contain `"EXASCHEMA=undefined"`

File: tools/run_tdvt_tests.sh (hunk 2 above)
Comment (kaklakariada) [reply tools/run_tdvt_tests.sh:6]: No. This is just a naming convention for the output / build directory.

**Review by MarleneKress79789 — APPROVED** [review]

**Pull request merged**